src/game/server/entities/character.cpp

```diff
@@ -45,6 +45,8 @@ CCharacter::CCharacter(CGameWorld *pWorld)
 : CEntity(pWorld, CGameWorld::ENTTYPE_CHARACTER, vec2(0, 0), ms_PhysSize)
 {
 	m_Health = 0;
```

Comment:
Love the change to lines 48 and 49. It'll be very useful for future projects where these values could change.